sdk/core/Azure.Core/api/Azure.Core.net5.0.cs

```diff
@@ -398,6 +400,15 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public abstract partial class DelayStrategy
+    {
+        protected DelayStrategy() { }
+        public static Azure.Core.DelayStrategy CreateExponentialDelayStrategy(System.TimeSpan? initialDelay = default(System.TimeSpan?), System.TimeSpan? maxDelay = default(System.TimeSpan?), double factor = 2, double minJitterFactor = 0.8, double maxJitterFactor = 1.2) { throw null; }
+        public static Azure.Core.DelayStrategy CreateFixedDelayStrategy(System.TimeSpan? delay = default(System.TimeSpan?)) { throw null; }
+        public static Azure.Core.DelayStrategy CreateSequentialDelayStrategy(System.Collections.Generic.IEnumerable<System.TimeSpan>? sequence = null) { throw null; }
+        public abstract System.TimeSpan GetNextDelay(Azure.Response? response, int retryNumber, System.TimeSpan? clientDelayHint, System.TimeSpan? serverDelayHint);
```

Comment:
Flip serverDelay and clientDelayHint order.

potentially introduce protected GetNextDelayCore(no hint parameters)
GetNextDelay would not be abstract => GetNextDelayCore.